src/main/g8/.scalafmt.conf

```diff
@@ -1,10 +1,10 @@
-version = 2.6.4
+version = 2.7.4
 
 style = defaultWithAlign
 
 docstrings                 = JavaDoc
 indentOperator             = spray
-maxColumn                  = 120
+maxColumn                  = 80
```

Comment:
why? 
Seems to narrow for 2021, no?

Reply:
You're right.

I just copy/pasted it from the [guide](https://github.com/akka/akka-platform-guide/blob/main/docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/.scalafmt.conf#L7). That width in the guide probably makes sense so snippets fit on the rendered HTML of the guide.

I'll revert to 120.